src/models/MainWeatherDetails.tsx

```diff
@@ -0,0 +1,23 @@
+export default class MainWeatherDetails {
+
+    temp: number;
+    pressure: number;
+    humidity: number;
+    tempMin: number;
+    tempMax: number;
+    seaLevel: number;
+    grndLevel: number;
+    tempKf: number;
```

Comment:
Same here. It is hard to find out what `tempKf` mean for me that is reading the code.